modules/storages/spec/lib/append_storages_hosts_to_csp_hook_spec.rb

```diff
@@ -52,12 +52,12 @@ def trigger_application_controller_before_action_hook
     hook_listener.application_controller_before_action(controller:)
   end
 
-  shared_examples 'appends storage host to CSP' do
-    it 'adds CSP directive to allow connection to storage host' do
+  shared_examples 'modifies CSP' do
```

Comment:
🟢 I tend to prefer the previous description as it was more specific _"appends storage host to CSP"_

Or we could name it like the method

```suggestion
  shared_examples 'content security policy directives' do
```

```ruby
include_examples 'content security policy directives'
```